content/guides/merchant-payment-conditions.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Merchant Payment Conditions
+---
+
+Some asset types such as tokens or closed-loop cards may require conditional operator approval. Merchant integrations are required to support [Payment Conditions](https://docs.centrapay.com/api/payment-requests#payment-condition) for these asset types in order for them to be accepted for payment.
+
+Examples of Merchant Payment Conditions include:
+
+- The merchant needs to confirm proof of identification for age-restricted purchases.
+- The merchant needs to confirm that a promotional item was purchased.
+- The patron needs to confirm the payment from their Centrapay integrated app.
+
+## Payment Conditions Flow
+
+In order to support Merchant Payment Conditions, the merchant integration must [create the Payment Request](https://docs.centrapay.com/api/payment-requests#create-a-payment-request) with the `conditionsEnabled` flag set to true.
```

Comment:
@MeganSteenkamp To resolve your other comment, we could say "the merchant integration must extend Centrapay's payment protocol by creating the Payment Request..."

Reply:
Thanks @tom-blake I have updated this in Notion and the PR.